app/enterprise/2.1.x/developer-portal/files-api.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Using the Portal Files API
+---
+
+## Portal Files API Overview
+
+The Portal Files API can be used as an alternative to the Portal CLI to manage
+dev portal content. Portal content must maintain a specific structure to render
```

Comment:
```suggestion
The Portal Files API is a small-scale alternative to the Portal CLI for managing Dev 
Portal content. Portal content must maintain a specific structure to render
```
Suggestion to make the scale distinction right away.

Reply:
This suggested doesn't seem to add value; rather, it seems to detract.  Prefer @Jlawlzz 's original description unless he states otherwise. Stet.

Reply:
Agree with @tlblessing, small-scale adds a correlation that doesn't necessarily exist.